packages/create-vite/src/index.ts

```diff
@@ -533,25 +611,31 @@ async function init() {
     setupReactSwc(root, template.endsWith('-ts'))
   }
 
-  let doneMessage = ''
-  const cdProjectName = path.relative(cwd, root)
-  doneMessage += `Done. Now run:\n`
-  if (root !== cwd) {
-    doneMessage += `\n  cd ${
-      cdProjectName.includes(' ') ? `"${cdProjectName}"` : cdProjectName
-    }`
-  }
-  switch (pkgManager) {
-    case 'yarn':
-      doneMessage += '\n  yarn'
-      doneMessage += '\n  yarn dev'
-      break
-    default:
-      doneMessage += `\n  ${pkgManager} install`
-      doneMessage += `\n  ${pkgManager} run dev`
-      break
+  if (immediate) {
+    if (!agent) throw new Error()
+    install(root, agent)
+    start(root, agent)
```

Comment:
Something I'm just wondering, maybe we should `cd` into the project so that when the dev server is stopped the directory is on the project, which also makes it feel like we're just auto-executing the commands we printed in non-immediate mode.

Reply:
I've took a stab but failed. Given that the shell is a parent process (the node process is the child process of that), it doesn't seem to be a way to tell the parent process to change the working directory.

Reply:
Shouldn't `process.chdir()` work here? I just tested it and it seems to work:

```js
import spawn from 'cross-spawn'

console.log(process.cwd())
console.log(spawn.sync('pwd').stdout.toString())
process.chdir('./src')
console.log(process.cwd())                       // prints new dir
console.log(spawn.sync('pwd').stdout.toString()) // also prints new dir
```